### ISSUE TITLE
Should support `requestFullscreen()`

### ISSUE THREAD

**Issue by jsantell** [google-ar/WebARonARKit#27]
via https://github.com/google-ar/three.ar.js/issues/24

**Comment by machenmusik** [comment]
Is the request for fullscreen camera view to avoid minor misalignment like #25, or to hide the browser chrome when fullscreen is requested (which #25 doesn't do), or both?

**Comment by joseeph** [comment]
@machenmusik I think I needed mainly for hiding the url bar to enable full screen experience. were there any solutions already? Many thanks. :p

**Comment by machenmusik** [comment]
Well, I had submitted a PR #25 that makes the webview full screen, but I am not sure @jsantell @Judax et al. are planning to merge.  Note that the PR does not wire up hiding the URL bar, which sounds like what you wanted to do

**Comment by machenmusik** [comment]
Right, so I think that #25 was a sub-optimal but convenient approach, but I think the first question is whether the minor misalignment (because the coordinate translation was not taking into account the controls chrome) is still happening
